sa/_db-next/migrations/20170718114448_DBCleanup.sql

```diff
@@ -0,0 +1,61 @@
+
+-- +goose Up
+-- SQL in section 'Up' is executed when this migration is applied
+DROP TABLE externalCerts;
+
+DROP TABLE identifierData;
+
+ALTER TABLE certificateStatus DROP LockCol;
+
+ALTER TABLE certificateStatus DROP subscriberApproved;
+
+START TRANSACTION;
+ALTER TABLE certificateStatus ADD id bigint(20) NOT NULL;
+ALTER TABLE certificateStatus DROP PRIMARY KEY, ADD PRIMARY KEY(id), ADD INDEX serial (serial);
+ALTER TABLE certificateStatus MODIFY COLUMN id bigint(20) NOT NULL AUTO_INCREMENT;
+COMMIT;
+
+START TRANSACTION;
+ALTER TABLE certificates ADD id bigint(20) NOT NULL;
+ALTER TABLE certificates DROP PRIMARY KEY, ADD PRIMARY KEY(id), ADD INDEX serial (serial);
+ALTER TABLE certificates MODIFY COLUMN id bigint(20) NOT NULL AUTO_INCREMENT;
+COMMIT;
+
+ALTER TABLE challenges DROP validated;
```

Comment:
Let's save this for another PR, since challenges is by far our hugest table. We probably actually just want to leave this alone for now, and wait for the authz reorganization.